lib/car.rb

```diff
@@ -0,0 +1,28 @@
+class Car
+  attr_reader :monthly_payment, :loan_length
+  attr_accessor :color
```

Comment:
:color should still be `attr_reader` since we only want to be able to write to that attribute via the `paint` method